javascript/javascript1/week1/homework1JS.js

```diff
@@ -0,0 +1,50 @@
+/*Age-ify 
+(A future age calculator)*/
+
+var yearOfBirth = 1995;
```

Comment:
```suggestion
const yearOfBirth = 1995;
```
It's better to use const în cases where the variable is not changed in the program